### PR TITLE
[release_candidate_changelog] CHANGELOG updates to prime for next RC.…

### PR DESCRIPTION
**Pull request by katelovescode**
… Needs @arangamani's review

### CODE REVIEW

**Comment by katelovescode** [comment]
@arangamani I'd also like to recommend we make develop the default branch so we can use git flow and build up stable code changes that will get merged to master only on version deploy, to prevent these concerns about merging to master in the future.  Thoughts?

**Comment by luke-hill** [comment]
From the comments raised and assuming every merge you've pushed in is correct. How easy is it to create a PR (Dependent on order of merges), of everything "UP TO" #192 ??

If you do that and release as a v1.6 at least it's a start, and we're then able to cut 192 and everything else as a 2.0 ?

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,19 @@ CHANGELOG
 
 upcoming
 --------
+Recommendation: Breaking change is `login_with_pry` config file - it's probable this will affect people's development or testing locally but shouldn't affect production usage; I'd be comfortable with this being 1.6.0, or bumping to 2.0.0 depending on @arangamani's thoughts.
```
[the remark this answers is not in the document]
Comment (katelovescode) [reply CHANGELOG.md]: There are a few that I can consider as candidates (purely because I like them and think they're cool :D ), but they will need to be tested and merged.  I'll add this to my to-do for next week.

[the remark this answers is not in the document]
Comment (katelovescode) [reply CHANGELOG.md]: After a brief skim of open PRs:
- I'd like to get #166 in, this line might be breaking because it changes what gets returned when creating a dumb slave, but I can look more closely to see if Jenkins is already returning true/false: https://github.com/arangamani/jenkins_api_client/pull/166/files#diff-e6faa48389a58b9287e126ac91aac0c7L259
- #190, #196, #199, #203, #205, #214, #237 are non-breaking and could be added as new functionality (they may need some documentation updates)

- #204 is definitely breaking and is important to add
- #206, #216 are bugfixes; if people have local workarounds they may be "breaking", but are worth adding

I'm not sure if #225 and #256 are ready; I'm also not sure if Groovy script is the right way to handle this, so I would like to leave those out for a later release.

Basically, given my choice, I'd like to incorporate everything except #225 and #256 if we're going to do a version update.  We can stop accepting PRs for the 2.0.0 release, and do it this way.  There are also a lot of unaddressed issues that I haven't had a chance to comment on yet; if you want more breaking changes and a bigger "strategy" I can spend some more time evaluating this and come up with a bigger release.  @arangamani - let me know what you think.

[the remark this answers is not in the document]
Comment (katelovescode) [reply CHANGELOG.md]: @arangamani - pinging again for visibility.

[the remark this answers is not in the document]
Comment (arangamani) [reply CHANGELOG.md]: @katelovescode Let's go with your proposal and bump version and include only the well-understood PRs.

[the remark this answers is not in the document]
Comment (katelovescode) [reply CHANGELOG.md]: @arangamani - I will review the open PRs listed above and merge them in if they work.  I'll update the changelog.  What's the process for a version bump?  Make a release in git tagged with the new version number and push to develop and master?

[the remark this answers is not in the document]
Comment (arangamani) [reply CHANGELOG.md]: Yeah, merge everything to master/develop, and then make a tag after all tests pass. Let me know when you have the tag ready, and I'll push it to rubygems.org. That part is still not automated yet.

[the remark this answers is not in the document]
Comment (katelovescode) [reply CHANGELOG.md]: Thanks!  Will do!
